src/app/routes/utils/regex/utils/index.js

```diff
@@ -179,7 +179,13 @@ export const getSecondaryColumnDataRegex = services => {
 };
 
 export const getIdxPageRegex = () => {
-  return `/:idx(${idxRegex}):amp(${ampRegex})?`;
+  let idxPageRegex = idxRegex;
+
+  if (isLive()) {
+    idxPageRegex = `${idxRegex}|persian/afghanistan`;
+  }
```

Comment:
Is this for compatibility to allow Live to point to the IDX pageType and Test to point to Topic pageType? Just wanting to confirm in my head along with https://github.com/bbc/simorgh/pull/11196/files#diff-b65c104a0afa7a7d406a9f367c3160e1c2d747f9280c595fa02b390ef153a994R8

Reply:
Correct - we only want to put this on the test environment for a couple of days to allow us to test it out. Then we're planning to migrate it completely next Tuesday, so it won't be for long.

Reply:
Cool, thanks!